test/src/596-catalogbalancer/main

```diff
@@ -0,0 +1,156 @@
+cvmfs_test_name="Catalog balancer"
+cvmfs_test_autofs_on_startup=false
+
+add_files() {
+  local repo_dir=$1
```

Comment:
Perhaps I don't understand this... further down you call for instance `add_files "$repo_dir/dir100"`.  So inside addfiles, `$repo_dir` is now a sub directory of the full path to the repository, no?